src/applications/check-in/pre-check-in/pages/Confirmation/index.jsx

```diff
@@ -1,11 +1,67 @@
-import React from 'react';
-import BackToHome from '../../components/BackToHome';
+import React, { useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
 
-export default function index() {
+import { focusElement } from 'platform/utilities/ui';
+import Telephone from '@department-of-veterans-affairs/component-library/Telephone';
+
+import AppointmentBlock from '../../components/AppointmentBlock';
+import { makeSelectVeteranData, makeSelectForm } from '../../selectors';
+
+const Confirmation = () => {
+  useEffect(() => {
+    focusElement('h1');
+  }, []);
+  const selectVeteranData = useMemo(makeSelectVeteranData, []);
+  const selectFormData = useMemo(makeSelectForm, []);
+  const { appointments } = useSelector(selectVeteranData);
+  const { data: formData } = useSelector(selectFormData);
+  const hasUpdates = Object.values(formData).includes('no');
   return (
-    <>
-      <h1>Confirmation</h1>
-      <BackToHome />
-    </>
+    <div
+      className="vads-l-grid-container vads-u-padding-bottom--3 vads-u-padding-top--3"
+      data-testid="confirmation-wrapper"
+    >
+      <h1 tabIndex="-1" className="vads-u-margin-top--2">
+        You’ve completed pre check-in
+      </h1>
+      <AppointmentBlock appointments={appointments} />
+      {hasUpdates === true ? (
+        <va-alert
+          background-only
+          status="info"
+          data-testid="confirmation-update-alert"
+        >
+          {/** TODO INFO ICON */}
+          <div>
+            A staff member will help you on the day of your appointment to
+            update your information.
+          </div>
+        </va-alert>
+      ) : (
+        <></>
+      )}
+      <p className={hasUpdates === true ? `vads-u-padding-left--2` : ``}>
```

Comment:
Super minor, but the `=== true` is unneeded.

Reply:
Updated in commit [1233dad](https://github.com/department-of-veterans-affairs/vets-website/pull/19570/commits/1233dad1e575ab03a669892a49d307c2801b2dbc)